src/swmm5.c

```diff
@@ -1050,8 +1050,8 @@ void getSemVersion(char* semver)
 //  
 //  NOTE: Each New Release should be updated in consts.h
 {
-	sprintf_s(semver, SEMVERSION_LEN, "%s.%s.%s", 
+	sprintf(semver, "%s.%s.%s", 
 		SEMVERSION_MAJOR, SEMVERSION_MINOR, SEMVERSION_PATCH);
```

Comment:
@bemcdonnell please explain the need use windows specific (non standard) methods, for asking the SWMM version...